modules/build.gradle

```diff
@@ -22,13 +22,13 @@ ext {
 	projectTemplateTokens = [
 		"biz.aQute.bnd.version": "3.5.0",
```

Comment:
I think we should bump the bnd to 4.2.0

Reply:
sure, I can do that. I can send a new PR tmrw when I remove `com.liferay.ant.bnd`